vpython/vpython.py

```diff
@@ -3138,17 +3138,15 @@ def pixel_to_world(self):
     def pixel_to_world(self, value):
         raise AttributeError('pixel_to_world is read-only')
 
-    def capture(self, *s):
-        if len(s) == 0:
-            raise AttributeError('scene.capture requires at least one argument.')
-        filename = s[0]
-        if not isinstance(filename, str): raise AttributeError('A capture file name must be a string.')
-        if '.png' not in filename: filename += '.png'
-        include_labels = "T"
-        if len(s) == 2:
-            if s[1] == True: include_labels = "T"
-            else: include_labels = "F"
-        self.addmethod('capture', include_labels+filename)
+    def capture(self, filename, capture_labels=True):
+        if not isinstance(filename, str):
+            raise TypeError("'filename' for Capture must be a string.")
```

Comment:
Very minor change: 

```suggestion
            raise TypeError("'filename' for capture must be a string.")
```

Reply:
therefore it's not capitalized

Reply:
That was the suggestion -- to change it from capitalized to not capitalized

Reply:
ok then, I'll resolve this conversation